InnerEye/ML/run_ml.py

```diff
@@ -59,16 +59,18 @@
     get_config_and_results_for_offline_runs, plot_cross_validation_from_files
 
 
-def try_to_mount_input_dataset() -> Optional[Path]:
+def try_to_mount_input_dataset(idx: int = 0) -> Optional[Path]:
     """
     Checks if the AzureML run context has a field for input datasets. If yes, the dataset stored there is
     returned as a Path. Returns None if no input datasets was found.
+
+    :param idx, suffix of AML dataset name, return path to INPUT_DATA_KEY_idx dataset
```

Comment:
malformed param docu? Also, idx is not a great variable name. dataset_index?

Reply:
updated